File: SWIFT/Cap08/c08ex03.swift
// c08ex03.swift

import Foundation

var Numeros: [Int] = Array(repeating: 0, count: 5)

print("CLASSIFICACAO DE NUMEROS (CRESCENTE)\n")

for i in 0...4 {
    print("Entre o \(i+1)o. numero: ", terminator: "")
    if let input = readLine() {
        Numeros[i] = Int(input) ?? 0
    }
}

for i in 0...3 {
    for j in i+1...4 {
        if (Numeros[i] > Numeros[j]) {
            let x = Numeros[i]
            Numeros[i] = Numeros[j]
            Numeros[j] = x
        }
    }
}

print()
for i in 0...4 {
    print("\(i+1)o. numero: \(Numeros[i])")
}

print()
print("Tecle <Enter> para encerrar... ", terminator: "")
_ = readLine()

File: KOTLIN/Cap08/c08ex03.kt
// c08ex03.kt

import java.util.*

fun main() {
    val numeros = IntArray(5)
    var x: Int

    println("CLASSIFICACAO DE NUMEROS (CRESCENTE)\n")

    for (i in 0..4) {
        print("${i + 1}o. numero: ")
        numeros[i] = readLine()?.toIntOrNull() ?: continue
    }

    for (i in 0..3) 
        for (j in i + 1..4)
            if (numeros[i] > numeros[j]) {
                x = numeros[i]
                numeros[i] = numeros[j]
                numeros[j] = x
            }

    println()
    for (i in 0..4)
        println("${i + 1}o. numero: ${numeros[i]}")

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
